GalaxyZooTouchTable/GalaxyZooTouchTable/Models/GalaxyExample.cs

```diff
@@ -8,47 +7,64 @@ public enum GalaxyType
-        public BitmapImage ImageThree { get; set; }
-        public string Title { get; set; }
-        public string Description { get; set; }
-        public string Description2 { get; set; }
+        BitmapImage ImageOne { get; }
+        BitmapImage ImageTwo { get; set; }
+        BitmapImage ImageThree { get; set; }
+        string Title { get; set; }
+        string Description { get; set; }
+        string Description2 { get; set; }
+    }
 
-        public GalaxyExample(GalaxyType type)
+    static class GalaxyExampleFactory
+    {
+        public static GalaxyExample Create(GalaxyType type)
         {
             switch (type)
             {
                 case GalaxyType.Smooth:
-                    ImageOne = new BitmapImage(new Uri("../Images/GalaxyExamples/smooth1.jpg", UriKind.Relative));
-                    ImageTwo = new BitmapImage(new Uri("../Images/GalaxyExamples/smooth2.jpg", UriKind.Relative));
-                    ImageThree = new BitmapImage(new Uri("../Images/GalaxyExamples/smooth3.jpg", UriKind.Relative));
-                    Title = "Smooth";
-                    Description = "Smooth galaxies fade gradually in all directions from the center";
-                    Description2 = "There may be a small bright symmetric core.";
-                    break;
+                    return new SmoothGalaxy();
                 case GalaxyType.Features:
-                    ImageOne = new BitmapImage(new Uri("../Images/GalaxyExamples/features1.jpeg", UriKind.Relative));
-                    ImageTwo = new BitmapImage(new Uri("../Images/GalaxyExamples/features2.jpg", UriKind.Relative));
-                    ImageThree = new BitmapImage(new Uri("../Images/GalaxyExamples/features3.jpg", UriKind.Relative));
-                    Title = "Features or Disk";
-                    Description = "Galaxies might have spiral arms, a bulge or bar, or any other interesting feature.";
-                    Description2 = "Choose this option if you see anything unique about the galaxy";
-                    break;
-                case GalaxyType.Star:
-                    ImageOne = new BitmapImage(new Uri("../Images/GalaxyExamples/star1.jpg", UriKind.Relative));
-                    ImageTwo = new BitmapImage(new Uri("../Images/GalaxyExamples/star2.jpg", UriKind.Relative));
-                    ImageThree = new BitmapImage(new Uri("../Images/GalaxyExamples/smooth3.jpg", UriKind.Relative));
-                    Title = "Star or Artifact";
-                    Description = "Choose \"Star or Artifact\" if there is no central galaxy to classify or if the artifact is so badly displayed that you can't ignore it and classify the galaxy with reasonable confidence.";
-                    Description2 = "The telescopes taking our data were designed to look at faint galaxies, which means that bright, compact objects like stars sometimes look a bit strange.";
-                    break;
+                    return new FeaturesGalaxy();
```

Comment:
These items were returned back to their factory settings so each example would have a unique class name, which is necessary when providing some of the animation logic in the individual styles under ExamplesPanel.xaml

Reply:
You couldn't just check the type value?

Reply:
I think this is the cleanest way to check with the animations on ExamplesPanel.xaml. The converter being used checks for the name of each object. Previously the name for all three objects was `GalaxyExample`

Reply:
fair enough!